src/frontend/packages/store/src/effects/user-favorites-effect.ts

```diff
@@ -144,4 +144,15 @@ export class UserFavoritesEffect {
       );
     })
   );
+
+  @Effect()
+  entityDeleteRequest$ = this.actions$.pipe(
+    ofType<EntityDeleteCompleteAction>(EntityDeleteCompleteAction.ACTION_TYPE),
+    mergeMap((action: EntityDeleteCompleteAction) => {
+      // Delete the favorite if there is one
+      this.store.dispatch(new RemoveUserFavoriteAction(action.asFavorite()));
```

Comment:
This results in a http request to delete a favourite regardless of whether the entity type is favouritable. For instance deleting an application with a route (through delete app stepper) results in two https requests (both succeed with `User Favorite deleted okay`).  I think we looked into this and there isn't a reliable way to determine if something's favourable (all places where an entity is favourited are manually applied).

Reply:
Yeah - we can revisit - its harmless at the moment, but we should add metadata on the entity catalog to say if an entity can be favorited.